test/e2e/tools/util.js

```diff
@@ -24,6 +24,10 @@ function testExists(testname) {
 }
 
 function rewriteTestFile(testname, testfile) {
+  if (testfile.indexOf('http') === 0) {
+    return testfile;
+  }
+
```

Comment:
Although this would match... `script="httpInterceptor.js"` Perhaps we should make it a little more complex: `/https?:\/\/`